www/app/app.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, OnInit } from '@angular/core';
+import { Http, Response } from '@angular/http';
+import { Observable } from 'rxjs/Rx';
+import { HTTP_PROVIDERS } from '@angular/http';
+
+@Component({
+  selector: 'my-app',
+  templateUrl: 'app/app.component.html',
+  styleUrls: ['app/app.component.css'],
+  providers: [ HTTP_PROVIDERS ]
+})
+
+export class AppComponent implements OnInit {
+  message: string = null;
+  currentStep: number = 0;
+  response: any;
+  previousCommand: string = '';
+  instructions: any;
+
+  constructor(private http: Http) { }
+
+  ngOnInit() {
+    this.getInstructions(this.currentStep);
+  }
+
+  evalInput(value) {
+    if (value.match(/^inspec exec\s*examples\/profile\/controls\/example.rb\s*/)) {
+      this.getResponse(value, "inspec_exec");
```

Comment:
This makes sense for the viability demo, but I think we might want to bundle the response data (or a segment of it) in a single blob that we send to the client to avoid a lot of round tripping.